src/_gettsim/config.py

```diff
@@ -99,6 +99,15 @@ def set_array_backend(backend: str):
         "description": "Spouses filing taxes jointly or individuals.",
         "potentially_endogenous": True,
     },
+    "ehe": {
```

Comment:
Just making a note here to see how often we'll need it in the end (apologies that my writing was confused over in #690). 

At least these days, it probably won't ever be relevant to work with aggregates at this level (sn/eg will be the relevant groupings). Might be different 20 years ago though.

The problem with it is that the aggregate is confusing for single persons when thinking about it intuitively. So there would be some value in doing without. Again, nothing to be done right now, just leaving this as a note.